packages/react-devtools-shared/src/devtools/views/Profiler/SnapshotSelector.css

```diff
@@ -28,3 +28,16 @@
   justify-content: center;
   color: var(--color-dim);
 }
+
+.Input {
+  font-size: var(--font-size-sans-normal);
+  text-align: right;
+  font-family: var(--font-family-monospace);
+  border: 1px solid transparent;
+  border-radius: 0.125rem;
+  padding: 0.125rem;
+}
+
+.Input:hover {
+  border: 1px solid var(--color-border);
+}
```

Comment:
I think this should probably have a :focus style similar to the editable value fields in props/state:
https://github.com/facebook/react/blob/87b3ada89dc83c88a89ff1b599b0f39e3c49cc32/packages/react-devtools-shared/src/devtools/views/Components/EditableValue.css#L31-L35

Probably also the same border radius:
https://github.com/facebook/react/blob/87b3ada89dc83c88a89ff1b599b0f39e3c49cc32/packages/react-devtools-shared/src/devtools/views/Components/EditableValue.css#L12